README.md

```diff
@@ -695,41 +693,62 @@ This feature requires a recent compiler with complete C++20 support:
 
 If you intend to use CMake's C++ module support (and possibly Ninja), then more recent tools are required:
 
-* Visual Studio 17.4 or later (providing `cl.exe` 19.34 or later)
-* Clang 16.0.0 or later
+* Visual Studio 2022 17.4 or later (providing `cl.exe` 19.34 or later)
+* Clang 17.0.0 or later
+* GCC 14.0 or later
 * CMake 3.28 or later
 * Ninja 1.10.2 or later
 
-Either way, GCC does not completely support C++ modules, is therefore not recommended for use.
-
 ##### Usage with CMake
 
-CMake is recommended for use with the Vulkan C++ module, as it provides a convenient platform-agnostic way to configure your project.
+CMake is recommended for use with the Vulkan-Hpp named module, as it provides a convenient platform-agnostic way to configure your project.
 CMake version 3.28 or later is required to support C++ modules.
 
 CMake provides the [FindVulkan module](https://cmake.org/cmake/help/latest/module/FindVulkan.html), which may be used to source the Vulkan SDK and Vulkan headers on your system.
-**Note that this module does not yet provide an IMPORTED target for the Vulkan C++ module, so you must set it up manually.**
 
-To use CMake with C++ modules, you must first set up `vulkan.cppm` as the source for a library target with `FILE_SET` configured  `TYPE = CXX_MODULES`, and then link it into your project.
+> [!WARNING]
+> The Vulkan-Hpp C++ named module is still experimental. Some suggested ways to use it in your projects are below. The long-term goal is to submit patches to the CMake [`FindVulkan`](https://cmake.org/cmake/help/latest/module/FindVulkan.html) module so that users may transparently configure the named module, without needing to declare it as an additional library in consumer CMake code.
 
 ```cmake
 # find Vulkan SDK
 find_package( Vulkan REQUIRED )
 
-# set up Vulkan C++ module
-add_library(VulkanCppModule)
-target_sources(VulkanCppModule PRIVATE
+# set up Vulkan C++ module as a library
+add_library( VulkanHppModule )
+target_sources( VulkanHppModule PRIVATE
   FILE_SET CXX_MODULES
+  BASE_DIRS ${Vulkan_INCLUDE_DIR}/vulkan
   FILES ${Vulkan_INCLUDE_DIR}/vulkan.cppm
 )

```

Comment:
Does this really work?
I think for me what really worked was
```
target_sources(VulkanHppModule PUBLIC
...
  **BASE_DIRS ${Vulkan_INCLUDE_DIR} 
  FILES ${Vulkan_INCLUDE_DIR}/vulkan/vulkan.cppm**
)```

Because the cppm file is in the vulkan folder.

Reply:
I just tested it—both work fine. Agreed that just using `${Vulkan_INCLUDE_DIR} is simpler, though...

Reply:
For me that line
`FILES ${Vulkan_INCLUDE_DIR}/vulkan.cppm`

needs to be
`FILES ${Vulkan_INCLUDE_DIR}/vulkan/vulkan.cppm`

if not I get this error

> [cmake] CMake Error at CMakeLists.txt:20 (target_sources):
[cmake]   Cannot find source file:
[cmake] 
[cmake]     C:/VulkanSDK/1.3.268.0/Include/vulkan.cppm
[cmake] 
[cmake] 
[cmake] CMake Error at CMakeLists.txt:18 (add_library):
[cmake]   No SOURCES given to target: VulkanHppModule

As you can see I am using the SDK (1.3.268.0) on windows . My Vulkan module is in `C:/VulkanSDK/1.3.268.0/Include/vulkan/vulkan.cppm`

What is your setup?

Reply:
Huh, that was a typo, and I changed the wrong line to boot. Proof that I shouldn't make commits at 2am.  

I also have `FILES ${Vulkan_INCLUDE_DIR}/vulkan/vulkan.cppm`; I'll change this right away.